.github/workflows/blocker-tracker-script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+SLACK_WEB_HOOK_URL=""
+
+JIRA_TOKEN=""
+
+function main() {
+    local blockers_keys=($(curl --request GET --url 'https://liferay.atlassian.net/rest/api/3/search?jql=project%20%3D%20%22PUBLIC%20-%20Liferay%20Product%20Delivery%22%20and%20labels%20%3D%20release-blocker%20and%20status%20%21%3D%20Closed&fields=issuekey' --user "jira-cloud-enterprisereleasehu@liferay.com:<$JIRA_TOKEN>" --header 'Accept: application/json' | jq -r '.issues[].key'))
+
+    if [ ! -d "/home/me/liferay-portal" ]
+    then
+       git clone git@github.com:liferay/liferay-portal.git
+    fi
+
+    cd liferay-portal || exit 3
```

Comment:
@kiwm, what are this exit code? Can you please clarify?

Reply:
In liferay-docker there are this exits codes: 
	LIFERAY_COMMON_EXIT_CODE_BAD=1
	LIFERAY_COMMON_EXIT_CODE_CD=3
	LIFERAY_COMMON_EXIT_CODE_HELP=2
	LIFERAY_COMMON_EXIT_CODE_MISSING_RESOURCE=5
	LIFERAY_COMMON_EXIT_CODE_OK=0
	LIFERAY_COMMON_EXIT_CODE_SKIPPED=4
In the code both the variable and the raw number is used if is needed I can create those constantes in the code I think this way will be better.